contrib/julia-config.jl

```diff
@@ -70,9 +70,9 @@ function cflags()
     arg2 = replace(includeDir(),"\\","\\\\")
     threading_def = threadingOn() ? "-DJULIA_ENABLE_THREADING=1 " : ""
     if is_unix()
-        return """$(threading_def)-fPIC -DJULIA_INIT_DIR=\\"$arg1\\" -I$arg2"""
+        return """-std=gnu99 $(threading_def)-fPIC -DJULIA_INIT_DIR=\\"$arg1\\" -I$arg2"""
```

Comment:
are we using gnu extensions in our headers, or which compiler versions don't default to c99 mode?